debian/changelog

```diff
@@ -1,3 +1,112 @@
+felix (2.4.0~__STREAM__) __STREAM__; urgency=low
+
+  * Felix 2.4.0 (from Git commit b891ac5).
+
+    A fully tested and production-ready Felix release, including the
+    changes from the following release candidates: 2.4.0-rc1, 2.4.0-rc2.
+
+    A summary of changes since Felix 2.3.0:
+
+    - Skip recalculation of selector matches if selector hasn't changed (#1482).
+    - Use updated Typha client API (#1484).
+    - Improve testing and test coverage (#1486, #1494, #1496, #1497).
+    - Make test suites produce junit reports (#1488).
+    - Allow selection of policy allow action (#1492).
+    - Implement liveness and readiness endpoints for Felix (#1489).
+    - Improve Calico version reporting (#1499).
+    - Streamline conntrack state deletions (#1500, #1498).
+    - Add release note to PR template (#1502).
+    - Add support for multiple CIDRs in a match rule (#1483, #1505).
+    - Support using a lock to coordinate iptables programming with other
+    - software (#1491, #1504).
+    - Move logutils functionality to libcalico-go (#1503).
+    - Add pre-DNAT policy support (#1506).
+    - Update glide pin for logrus (#1509).
+    - Allow for time fuzziness in route table UT (#1510).
+    - Update to Typha v0.3.0 (#1512).
+    - Only report ready after first apply() completes.
```

Comment:
I'll try to find and add the PR numbers.
Think it also makes sense - given that I'm manually editing this - to remove the duplication, i.e. the 2.4.0-rc1 and 2.4.0-rc2 sections - agree?